apps/web/app/error.tsx

```
@@ -3,12 +3,15 @@
 // Error components must be Client components
 import { Button } from "@/modules/ui/components/button";
 import { ErrorComponent } from "@/modules/ui/components/error-component";
+import * as Sentry from "@sentry/nextjs";
 import { useTranslate } from "@tolgee/react";
 
 const Error = ({ error, reset }: { error: Error; reset: () => void }) => {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)